src/text-link.css.ts

```diff
@@ -0,0 +1,58 @@
+import {style} from '@vanilla-extract/css';
+import * as mq from './media-queries.css';
+import {vars} from './skins/skin-contract.css';
+
+export const textLink = style({
+    width: 'auto',
+    lineHeight: 'inherit',
+    display: 'inline',
+    color: vars.colors.textLink,
+    wordBreak: 'break-word',
+    cursor: 'pointer',
+    '@media': {
+        [mq.supportsHover]: {
+            selectors: {
+                '&:hover:not([disabled])': {
+                    textDecoration: 'underline',
+                },
+            },
+        },
+    },
+    selectors: {
+        '&[disabled]': {
+            opacity: 0.5,
+            textDecoration: 'none',
+        },
+    },
+});
```

Comment:
we have something called `[sprinkles](https://vanilla-extract.style/documentation/packages/sprinkles/)` for styles that are widely used in mistica, so here you must combine sprinkles with styles, something like 

```
style([
    sprinkles({width: 'auto', display: 'inline', color: vares.colors.textLink}),
   { ...rest of styles...}
])
```


you can check sprinkles.css.ts to check what styles do we have here



also apply this change for the rest of the styles

Reply:
Thank you! I was aiming to learn about `vanilla-extract`, so all your feedback is very welcome.
I'll look into that!